tox.ini

```diff
@@ -6,13 +6,9 @@ skipsdist = true
 
 [testenv:docs]
 basepython = python3
-deps = -r{toxinidir}/requirements.txt
+deps = -r{toxinidir}/test-requirements.txt
```

Comment:
Just curious: what's the semantic difference between test-requirements and requirements here?

Reply:
@gkunz: requirements.txt is for package dependencies (especially in case of PBR which defacto reads it). Here it's just about dependencies needed for building docs ; no package at all. Yes maybe too much precise :)

Reply:
ok, thanks! Subtle difference.